src/prefect/cli/deploy.py

```diff
@@ -80,6 +82,123 @@
 )
 
 
+@app.command()
+async def init(
```

Comment:
After removing deprecated cmds, the single remaining cmd from `prefect.cli.project.py` is moved into `prefect.cli.deploy.py`